google/resource_storage_transfer_job.go

```diff
@@ -320,6 +320,13 @@ func gcsDataSchema() *schema.Resource {
 				Type:        schema.TypeString,
 				Description: `Google Cloud Storage bucket name.`,
 			},
+			"path": {
+				Optional:     true,
+				Type:         schema.TypeString,
+				Description:  `Google Cloud Storage path in bucket to transfer`,
+				Default:      "",
```

Comment:
we should remove the default value here. Ideally we won't send this value if not set by the user. 

If the value is returned by the api after creation use `Computed: true`